recipes/reframed/meta.yaml

```diff
@@ -43,7 +45,5 @@ about:
 
 extra:
   recipe-maintainers:
-    - acaprez
-  identifiers:
-    - biotools:reframed
-    - doi:10.5281/zenodo.7955995
+    - lhtxa
+    - cdanielmachado
```

Comment:
Is the person you add here fine with being added as a maintainer?

Reply:
Yes, the new maintainer I added is the developer of reframed and is fine with being added as a maintainer here.